elastic-agent-client.proto

```diff
@@ -323,10 +323,12 @@ message UnitObserved {
 message CheckinObservedVersionInfo {
     // Name of the binary.
     string name = 1;
-    // Version of the binary.
+    // Agent package version sent by the agent.
     string version = 2;
     // Additional metadata about the binary.
     map<string, string> meta = 3;
+    // VCS commit hash of the binary.
+    string build_hash = 4;
```

Comment:
Since this has the changes from https://github.com/elastic/elastic-agent-client, shouldn't it be targeted to merge to the AndersonQ:beats-report-package-version branch so we don't see both changes in the PR?

Reply:
If I'd have done so, the PR would be on my fork only, not here.